lib/tasks/migrate_paper_trail.rake

```diff
@@ -0,0 +1,19 @@
+require 'paper_trail/frameworks/active_record'
+
+desc "Migrate PaperTrail YAML columns to JSON"
+task migrate_paper_trail: :environment do
+  # https://github.com/paper-trail-gem/paper_trail/blob/v12.3.0/README.md#convert-existing-yaml-data-to-json
+  most_recent_record = PaperTrail::Version.last.id
+
+  PaperTrail::Version.where.not(old_object: nil).find_each do |version|
```

Comment:
I would have liked to do this in one with `PaperTrail::Version.where.not(old_object: nil).where.not(old_object_changes: nil)`, but that killed the console.